drivers/video/fbdev/ssd1307fb.c

```diff
@@ -249,6 +250,65 @@ static int ssd1307fb_blank(int blank_mode, struct fb_info *info)
 		return ssd1307fb_write_cmd(par->client, SSD1307FB_DISPLAY_ON);
 }
 
+static int ssd1307fb_set_par(struct fb_info *info)
+{
+	struct ssd1307fb_par *par = info->par;
+	int ret;
+  u32 com_invdir;
+
+	/* Set segment re-map */
+	if (par->seg_remap) {
+		ret = ssd1307fb_write_cmd(par->client, SSD1307FB_SEG_REMAP_ON);
+		if (ret < 0)
+			return ret;
+	};
+
+	/* Set COM direction */
+	com_invdir = 0xc0 | (par->com_invdir & 0x1) << 3;
+	ret = ssd1307fb_write_cmd(par->client,  com_invdir);
+	if (ret < 0)
+		return ret;
+
+	/* Switch to horizontal addressing mode */
+	ret = ssd1307fb_write_cmd(par->client, SSD1307FB_SET_ADDRESS_MODE);
+	if (ret < 0)
+		return ret;
```

Comment:
Consider:
`
 	/* Switch to horizontal addressing mode */
	if (ret = ssd1307fb_write_cmd(par->client, SSD1307FB_SET_ADDRESS_MODE) < 0)
		return ret;
`